clojure-lang/README.md

```diff
@@ -0,0 +1,8 @@
+# clojure-lang
+
+Code examples implemented in Clojure.
+
+## Installation
+
+Install Clojure using [Leiningen](https://leiningen.org/).
```

Comment:
It might be better to point people off to the official "getting started" guide for Clojure, https://clojure.org/guides/getting_started, and let them pick Leiningen if they want it.

Reply:
@josiahcarlson Done.